Behavior/Resources/Localizable.xcstrings

```diff
@@ -338,6 +340,7 @@
       }
     },
     "NOTIFICATION_PERMISSIONS_SUBTITLE" : {
+      "extractionState" : "stale",
```

Comment:
Please remove all stale localization values.

If you want to learn more about localization, please check out this video for more details: https://developer.apple.com/wwdc23/10155.